examples/wordpress/README.md

```diff
@@ -42,7 +42,7 @@ The template contains the following variables:
 
 ```
 {% set PROPERTIES = properties or {} %}
-{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-prod' %}
+{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-testing' %}
```

Comment:
These changes were made after the code freeze and became a rebasing issue.  There is comment describing the issue in this thread.  :point_up: that way, I think.